### PR TITLE
Temp/category

### PR DESCRIPTION
**Pull request by Yshah112**

### CODE REVIEW

File: products/forms.py [hunk 1]
```diff
@@ -1 +1,79 @@
 from django import forms
+from .models import Product
+from .models import Category
+from .models import Brand
```
Comment (palakparmartntra) [inline products/forms.py]: import in one line, if all the models are in one file. @Yshah112 @rinkishivani

File: products/views.py [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+from .forms import AddBrandForm, UpdateBrandForm
+from django.http import Http404
+from django.shortcuts import render, redirect, get_object_or_404
+from django.contrib import messages
+from .messages import BrandFormSuccessMessages
+from .messages import BrandFormErrorMessages
+from django.contrib.auth.decorators import login_required
+from products.exceptions import CannotDeleteBrandException
+from .headings import AdminPortalHeadings
+from .forms import AddProductForm
```
Comment (palakparmartntra) [inline products/views.py]: here as well @Yshah112 @rinkishivani

File: products/forms.py (hunk 1 above)
Comment (Yshah112) [reply products/forms.py]: done

File: products/views.py (hunk 2 above)
Comment (Yshah112) [reply products/views.py]: done

File: products/forms.py [hunk 3]
```diff
@@ -1 +1,77 @@
...
+
+class AddProductForm(forms.ModelForm):
+    class Meta:
+        model = Product
+        fields = ['name', 'description', 'quantity', 'price', 'image', 'category', 'is_active']
+
+
+class AddCategoryForm(forms.ModelForm):
+    class Meta:
+        model = Category
```
Comment (pratikbariya) [inline products/forms.py:16]: @Yshah112 add doc string for this form class

File: products/urls.py [hunk 4]
```diff
@@ -1,7 +1,30 @@
 from django.urls import path
 from django.conf.urls.static import static
 from django.conf import settings
+from .views import home_page
+from .views import add_product, update_product, view_product, delete_product, \
+    trash_product, soft_delete, restore
+from .views import add_category, update_category, view_categroy, delete_category
```
Comment (pratikbariya) [inline products/urls.py]: @Yshah112 if there are many imports then use tuple. like: from .views import (add_product, update_product, view_product, delete_product,  trash_product, soft_delete, restore)

File: products/views.py [hunk 5]
```diff
@@ -0,0 +1,371 @@
+
+from django.db.models import Count
+from django.db.models import Q
+from .models import Category, Brand, Product
```
Comment (pratikbariya) [inline products/views.py:3]: @Yshah112 use single import for Count and Q

File: products/forms.py (hunk 3 above)
Comment (Yshah112) [reply products/forms.py:16]: done

**Review by pratikbariya — APPROVED** [review]

**Pull request merged**